docs/api.rst

```diff
@@ -1335,7 +1418,9 @@ newDateClock
 
   newDateClock :: () -> Clock
 
-Create a new :ref:`Clock` using`Date.now`. Note that a Clock using `Date.now` is not guaranteed to be monotonic, and is subject to system clock changes, e.g. NTP can change your system clock!
+Create a new :ref:`Clock` using `Date.now`. Note that a Clock using `Date.now` 
```

Comment:
Oh man, my markdown muscle memory was in full swing: double backticks for these two `Date.now`s.